index.html

```diff
@@ -1,15 +1,15 @@
 <!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8">
+<!-- saved from url=(0039)https://skycocker.github.io/chromebrew/ -->
+<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <link rel="shortcut icon" href="brew.png"" type="image/png">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="description" content="The missing package manager for Chrome OS">
     <meta name="author" content="Michal Siwek">
-    <link rel="shortcut icon" href="favicon.png">
+    <link rel="shortcut icon" href="https://skycocker.github.io/chromebrew/favicon.png">
 
     <title>Chromebrew - the software source you missed so much</title>
 
-    <link href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css" rel="stylesheet">
+    <link href="bootstrap.min.css" rel="stylesheet">
     <link href="jumbotron-narrow.css" rel="stylesheet">
     <link href="bs-example.css" rel="stylesheet">
     <link href="crew-specific.css" rel="stylesheet">
```

Comment:
Can these `css` files be put in a `css` folder and referenced via `<link href="/css/bootstrap.min.css" rel="stylesheet">`?

Reply:
sure

Reply:
All of them?